packages/vscode-extension/src/commands/UpdateObject.ts

```diff
@@ -31,19 +33,42 @@ export default class UpdateObject extends BaseAction {
 
 		const continueMessage = this.translationService.getMessage(ANSWERS.CONTINUE);
 		const cancelMessage = this.translationService.getMessage(ANSWERS.CANCEL);
-		const override = await window.showQuickPick([continueMessage, cancelMessage], {
-			placeHolder: this.translationService.getMessage(UPDATE_OBJECT.OVERRIDE, scriptId),
+		const yes = this.translationService.getMessage(ANSWERS.YES);
+		const no = this.translationService.getMessage(ANSWERS.NO);
+		const processCanceledMessage = this.translationService.getMessage(UPDATE_OBJECT.PROCESS_CANCELED);
+
+		const overwriteObject = await window.showQuickPick([continueMessage, cancelMessage], {
+			placeHolder: this.translationService.getMessage(UPDATE_OBJECT.OVERWRITE, scriptId),
 			canPickMany: false,
 		});
 
-		if (!override || override === this.translationService.getMessage(ANSWERS.CANCEL)) {
-			this.messageService.showInformationMessage(this.translationService.getMessage(UPDATE_OBJECT.PROCESS_CANCELED));
+		if (!overwriteObject || overwriteObject === cancelMessage) {
+			this.messageService.showInformationMessage(processCanceledMessage);
 			return;
 		}
 
+		const includeInstancesAnswer = await window.showQuickPick([yes, no], {
+			placeHolder: this.translationService.getMessage(UPDATE_OBJECT.QUESTIONS.INCLUDE_INSTANCES),
+			canPickMany: false,
+		});
+
+		let includeInstancesFlag = '';
+		if (includeInstancesAnswer === yes) {
+			const overwriteInstances = await window.showQuickPick([continueMessage, cancelMessage], {
```

Comment:
I'm not in favor of asking twice about overwriting, once for regular objects, and again for the instances of custom records. I think is a bad user experience for the developer. I would prefer to ask only once, either overwrite objects or overwrite objects and its instances.
I'm actually not sure that we could update more than one object at a time, but I would like to be only be asked about overwriting only once in any case.